crates/edr_provider/src/data.rs

```diff
@@ -95,11 +97,22 @@ pub struct ProviderData {
     last_filter_id: U256,
     logger: Logger,
     impersonated_accounts: HashSet<Address>,
+    callbacks: Arc<dyn SendInspectorCallbacks>,
 }
 
+pub trait InspectorCallbacks {
+    /// Calls to `console.*` from Solidity
+    fn console(&self, call_input: Bytes);
+}
+
+pub trait SendInspectorCallbacks: InspectorCallbacks + Debug + Send + Sync {}
```

Comment:
```suggestion
pub trait SyncInspectorCallbacks: InspectorCallbacks + Debug + Send + Sync {}
```

Other instances of these super traits are called `SyncBlock`, `SyncBlockchain`, etc.

Reply:
Good catch, fixed in [9ad0b10](https://github.com/NomicFoundation/hardhat/pull/4659/commits/9ad0b10c7bae521f42179af648304be514289149). I think it only had a `Send` bound originally, hence the wrong name.